packages/flutter/lib/src/services/restoration.dart

```diff
@@ -529,6 +524,16 @@ class RestorationBucket extends ChangeNotifier {
   RestorationManager? _manager;
   RestorationBucket? _parent;
 
+  /// Returns true when entities processing this bucket should restore their
+  /// state from the information in the bucket (e.g. via [read] and
+  /// [claimChild]) instead of copying their current state information into the
+  /// bucket (e.g. via [write] and [adoptChild].
+  ///
+  /// This flag is true for one frame after the [RestorationManager] has been
```

Comment:
nit: It's unlikely, but the "is true for one frame after" could be misconstrued as "is true for one of the frames after" (i.e. not just the immediately following frame, but one of any of the following frames).

Maybe say "is true for the frame after" instead? Here and above.